app/views/runs/_timing_badge.slim

```diff
@@ -0,0 +1,16 @@
+- if run.completed?(Run::REAL) && run.completed?(Run::GAME)
```

Comment:
It's a bit hard to find the timing switch if you don't know where to look for it.  Not really sure a great fix, but Stef also agreed that it's not as obviously as to where it is.